src/harness/harnessIO.ts

```diff
@@ -786,21 +786,20 @@ export namespace Compiler {
 
             if (!isSymbolBaseline) {
-                const perfStats: [name: string, reportThreshold: number, rounding: number, beforeValue: number, afterValue: number][] = [];
-                perfStats.push(["Strict subtype cache", 1000, 100, prePerformanceValues.strictSubtype, postPerformanceValues.strictSubtype]);
-                perfStats.push(["Subtype cache", 1000, 100, prePerformanceValues.subtype, postPerformanceValues.subtype]);
-                perfStats.push(["Identity cache", 1000, 100, prePerformanceValues.identity, postPerformanceValues.identity]);
-                perfStats.push(["Assignability cache", 1000, 100, prePerformanceValues.assignability, postPerformanceValues.assignability]);
-                perfStats.push(["Type Count", 1000, 100, prePerformanceValues.typeCount, postPerformanceValues.typeCount]);
-                perfStats.push(["Instantiation count", 1500, 500, prePerformanceValues.instantiation, postPerformanceValues.instantiation]);
-                perfStats.push(["Symbol count", 45000, 500, prePerformanceValues.symbol, postPerformanceValues.symbol]);
-
-                if (perfStats.some(([, threshold, , , postValue]) => postValue >= threshold)) {
+                const perfStats: [name: string, reportThreshold: number, beforeValue: number, afterValue: number][] = [];
+                perfStats.push(["Strict subtype cache", 1000, prePerformanceValues.strictSubtype, postPerformanceValues.strictSubtype]);
+                perfStats.push(["Subtype cache", 1000, prePerformanceValues.subtype, postPerformanceValues.subtype]);
+                perfStats.push(["Identity cache", 1000, prePerformanceValues.identity, postPerformanceValues.identity]);
+                perfStats.push(["Assignability cache", 1000, prePerformanceValues.assignability, postPerformanceValues.assignability]);
+                perfStats.push(["Type Count", 1000, prePerformanceValues.typeCount, postPerformanceValues.typeCount]);
+                perfStats.push(["Instantiation count", 1500, prePerformanceValues.instantiation, postPerformanceValues.instantiation]);
+                perfStats.push(["Symbol count", 45000, prePerformanceValues.symbol, postPerformanceValues.symbol]);
+
+                if (perfStats.some(([, threshold, , postValue]) => postValue >= threshold)) {
                     perfLines.push(`=== Performance Stats ===`);
-                    for (const [name, _, rounding, preValue, postValue] of perfStats) {
-                        const preDisplay = valueToString(preValue, rounding);
-                        if (preDisplay !== "0") {
-                            perfLines.push(`${name}: ${preDisplay} / ${valueToString(postValue, rounding)} (nearest ${rounding})`);
+                    for (const [name, threshold, preValue, postValue] of perfStats) {
+                        if (postValue >= threshold) {
+                            perfLines.push(`${name}: ${valueToString(preValue)} / ${valueToString(postValue)}`);

```

Comment:
This is a nit but could you (while here) change this to `->` instead of `/`? I had to explain to a few people that this was a before/after and not a percentage or something.

Reply:
Thought - if the post-value is identical, maybe don't show it at all. If it's different maybe just write out `(post-check: #####)`

Reply:
Done and done